Adding deprecation warning for Interactive Users using ApiKeys (#136422)
elastic/kibana: docs/user/security/api-keys/index.asciidoc

--header 'kbn-xsrf: true' \
--header 'Authorization: ApiKey aVZlLUMzSUJuYndxdDJvN0k1bU46aGxlYUpNS2lTa2FKeVZua1FnY1VEdw==' \

[IMPORTANT]
============================================================================

> API keys are intended for programmatic access to {kib} and {es}, and should not be used to authenticate access via a web browser.

Reviewer: Let's go with Gail's suggested tweak of my wording:

```suggestion
API keys are intended for programatic access to {kib} and {es). Do not use API keys to authenticate access via a web browser.
```
